src/lib/global.ts

```diff
@@ -1,46 +1,28 @@
 import { reactive } from "vue";
 import getLang from "../loc/lang"
-import { Client, ClientType } from "src/lib/client"
+import { Client } from "src/lib/client"
 import { AutoLauncher } from "src/lib/native"
 
-
 let text: { [index: string]: { [index: string]: string } } = {}
-export type GlobalMutations = {
-  initClient(): void
-  initLaunchOnBoot(): void
-  changeLang(newLang: string): void
-  loadLangData(): void
-}
-
 export class Global {
-  private client!: ClientType
-  data = reactive({ status: { state: "loading", message: "loading" }, loc: { selected: 'en', text }, launchOnBoot: new AutoLauncher, client: this.client })
-  mutations: GlobalMutations = {
-    initClient: this.initClient,
-    initLaunchOnBoot: this.initLaunchOnBoot,
-    changeLang: this.changeLang,
-    loadLangData: this.loadLangData
-  }
-  protected async initClient() {
-    const client = await Client()
-    this.data.client = client
-  }
-  protected async initLaunchOnBoot() {
-    this.data.launchOnBoot.appName = 'app'
-    await this.data.launchOnBoot.init()
-  }
-  protected async changeLang(newLang: string) {
+  client = new Client
```

Comment:
Add big TODO to refactor this to either protect invariants (it exposes `.client` that is in a broken state until you call `.client.init()` and `await` on it or better yet refactor completely and use dependency injection, such that users of this don't even run before client is ready to accept requests.